base/tps/src/org/dogtagpki/server/tps/processor/TPSProcessor.java

```diff
@@ -105,7 +107,7 @@
 import com.netscape.certsrv.tps.token.TokenStatus;
 import com.netscape.cms.logging.Logger;
 import com.netscape.cms.logging.SignedAuditLogger;
-import com.netscape.cms.servlet.tks.SecureChannelProtocol;
+//import com.netscape.cms.servlet.tks.SecureChannelProtocol;
```

Comment:
Should we just remove it completely? :-)

Reply:
Yes, I will take that out too. Thx.